server/form-pages/apply/basic-information/oralHearing.ts

```diff
@@ -1,12 +1,23 @@
-import type { TasklistPage, ObjectWithDateParts } from 'approved-premises'
+import type { TasklistPage, ObjectWithDateParts, YesOrNo } from 'approved-premises'
 import { dateAndTimeInputsAreValidDates } from '../../../utils/utils'
 
 export default class OralHearing implements TasklistPage {
   name = 'oral-hearing'
 
   title = 'Do you know Robert Brown’s oral hearing date?'
 
-  constructor(readonly body: Record<string, unknown>) {}
+  body: ObjectWithDateParts<'oralHearingDate'> & {
+    knowOralHearingDate: YesOrNo
+  }
+
+  constructor(body: Record<string, unknown>) {
```

Comment:
I don't understand the advantage of typing the properties on this object to be`unknown` if we're then going to immediately say they're strings/YesOrNo. Unless I'm missing something I think it would be more readable to have `Record<string, string | YesOrNo>` here

Reply:
I think my thinking was we literally don't know what is being passed into the constructor, so it makes sense to be flexible

Reply:
Yeah after having some time to reflect I think it does make sense to me. 
We're saying 'We don't know what the properties object are but if it has *x* property then we know it's *y* type'